containers/logs/LogsSection.js

```diff
@@ -25,16 +25,16 @@ import { ELEMENTS_PER_PAGE, LOGS_STATE } from 'proton-shared/lib/constants';
 import LogsTable from './LogsTable';
 import WipeLogsButton from './WipeLogsButton';
 
-const EVENTS = {
-    0: c('Log event').t`Login password failure`,
-    1: c('Log event').t`Login success`,
-    2: c('Log event').t`Logout`,
-    3: c('Log event').t`2FA login failure`
-};
-
 const { DISABLE, BASIC, ADVANCED } = LOGS_STATE;
 
 const LogsSection = () => {
+    const EVENTS = {
+        0: c('Log event').t`Login password failure`,
```

Comment:
Can we also use constants here please.

Reply:
Yeah, actually those constants are already defined. It seems there is an extra event, LOGIN_SUCCESS_AWAIT_2FA. I created a description of the event
"Login password success, awaiting 2FA"
but I'm not sure that's what it means.